public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js

```diff
@@ -3339,3 +3339,145 @@ YetiForce_Widget_Js(
 		}
 	}
 );
+YetiForce_Widget_Js(
+	'YetiForce_TimeCounter_Widget_Js',
+	{},
+	{
+		hr: 0,
+		min: 0,
+		sec: 0,
+		counter: true,
+		timeStart: '',
+		timeStop: '',
+		/**
+		 * Show quick create form
+		 */
+		postLoadWidget: function () {
+			this._super();
+			this.registerNavigatorButtons();
+		},
+		/**
+		 * Register events on the navigation buttons.
+		 */
+		registerNavigatorButtons: function () {
+			const container = this.getContainer();
+			let instance = this;
+			let btnStart = container.find('.js-start-watch');
+			let btnStop = container.find('.js-stop-watch');
+			let btnReset = container.find('.js-reset-watch');
+			let navigator = container.find('.js-navigator');
```

Comment:
Correction of the class name to be more specific for this widget. The current names are too general